### DISCUSSION TITLE
809

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#984] [Comments]
# 809

지난주 토요일에 KSUG 에서 진행한 가을 세미나의 패널토크 MC로 참여하고 왔다.기완님, 희송님, 용권님 3분이 여러 질문에 대해 답변을 해주시는 것에 많은 경험을 간접적으로 얻을 수 있었다.이런 패널토크는 사전에 준비된 질문 외에도 갑작스런 질문들이 많은데, 준비되지 않은 내용에 대해서 바로 바로 이야기하려면 그만큼 여러 경험이나 생각이 정리되어있어야만 한다고 생각했다.그래서 3분의 생각이 듣고 싶어 사전에 논의된 질문 외에도 여러가지 추가 질문들을 이어서 했고, 다채로운 경험을 얻을 수 있었다.끝나고 와이프를 만나러 가는 길에 웬지 스토브리그 드라마의 장면이 생각났다.스토브리그에서 단장인 백승수가 드림즈가 준우승 했을 때와 지금이 무엇이 다른지를 당시의 멤버였던 장진우 투수에서 물어보는 장면이 있다..

https://jojoldu.tistory.com/809

**Comment by devmeeple** [comment]
향로님께서 평소 읽으시는 책이나 시청했던 영상을 적절하게 인용해서
메시지 전달해 주셔서 감사합니다.

최근, '암묵지와 직관은 어떻게 수련할까'에 대해 생각하고 있었는데
'형식지 수련'은 또 다른 사고네요.

재밌게 읽었습니다.
